gulpfile.js

```diff
@@ -0,0 +1,22 @@
+var gulp     = require('gulp'),
+    mocha    = require('gulp-mocha'),
+    babel    = require('babel'),
+    plumber  = require('gulp-plumber'),
+    istanbul = require('gulp-istanbul');
```

Comment:
I _feel_ like its less visually noisy. I do think it can be a pain for the reasons you state, but I normally use either anyway for readability.